apps/block_scout_web/test/block_scout_web/controllers/api/v2/address_controller_test.exs

```diff
@@ -1710,6 +1710,25 @@ defmodule BlockScoutWeb.API.V2.AddressControllerTest do
       assert response_1 == response
     end
 
+    # retrieve only fee payments with mainchain reward using value_from_mainchain query parameter
+    test "get fee payments with mainchain reward", %{conn: conn} do
+      address = insert(:address)
```

Comment:
To also check that the address filtering works as expected, you can insert another address, `address_two`, and another `fee_payment_address_two` and make sure 6 and 3 items are returned respectively

Reply:
done